ui/packages/registry-admin-local-ui/src/main/webapp/js/model/Field.js

```diff
@@ -157,9 +157,9 @@ define([
                     }
                 } else if (type === 'date') {
                     if (!moment(this.get('valueDate')).isValid()) {
```

Comment:
This whole file (and the rest of the registry UI code) really needs to be refactored but for now it works. I just made the most minimal change I could here to avoid risking breaking other things.

Reply:
The back end handles most all fields as generic strings and doesn't know/care if it is a date or not so a bad date will not hurt the backend. If a bad date does get into a "date" slot the ui does the same kind of validation check first  before trying to use the information as a date. We ran into this early on when someone injected an ebrim message into the catalog with an invalid date  in a slot. The only issue it caused was in the ui and that was fixed.